docs/source/config-configuration.md

```diff
@@ -215,7 +215,7 @@ Sector coupling studies are all under active development. More info to come!
 (clustering_cf)=
 ## `clustering`
 
-When clustering `aggregation_zones` defines the region boundaries which will be respected through the clustering process; State boarders, balancing authority regions, or REeDs shapes. This feature is important for imposing constraints (`opts`) which are defined over specific regions. For example, the data included in the model on interface transfer capacities are prepared for REeDs shapes but not states and BA regions. Moving forward we plan to use REeDs shapes as our default however we will maintain States and BA regions as well.
+When clustering `topological_boundaries` defines the region boundaries which will be respected through the clustering process; State boarders, balancing authority regions, or REeDs shapes. This feature is important for imposing constraints (`opts`) which are defined over specific regions. For example, the data included in the model on interface transfer capacities are prepared for REeDs shapes but not states and BA regions. Moving forward we plan to use REeDs shapes as our default however we will maintain States and BA regions as well.
```

Comment:
is balancing authorities supported? Config file only shows state and reeds_zone in the comment

Reply:
Also the ValueError in build_bus_regions says something different

```
ValueError: zonal_aggregation must be either balancing_area, country, reeds_id, or state
```

Reply:
removing this so we can move forward with only reeds_zone and county supported.